Build issue

I am getting following cmake build out


```

Microsoft (R) Build Engine version 16.4.0+e901037fe for .NET Framework
Copyright (C) Microsoft Corporation. All rights reserved.

  main.cpp
D:\ws\pybind\carma\include\carma/carma/converters.h(542,25): error C2882: 'pybind11': illegal use of namespace identifi
er in expression [D:\ws\pybind\build\cmake_example.vcxproj]
D:\ws\pybind\carma\include\carma/carma/converters.h(542,25): error C4430: missing type specifier - int assumed. Note: C
++ does not support default-int [D:\ws\pybind\build\cmake_example.vcxproj]
D:\ws\pybind\carma\include\carma/carma/converters.h(543,1): error C2146: syntax error: missing ';' before identifier 'N
AMESPACE_BEGIN' [D:\ws\pybind\build\cmake_example.vcxproj]
D:\ws\pybind\carma\include\carma/carma/converters.h(546,61): error C2065: 'armaT': undeclared identifier [D:\ws\pybind\
build\cmake_example.vcxproj]
D:\ws\pybind\carma\include\carma/carma/converters.h(546,46): error C2923: 'carma::is_convertible': 'armaT' is not a val
id template type argument for parameter 'T' [D:\ws\pybind\build\cmake_example.vcxproj]
D:\ws\pybind\carma\include\carma/carma/converters.h(546,69): error C2955: 'carma::is_convertible': use of class templat
e requires template argument list [D:\ws\pybind\build\cmake_example.vcxproj]
D:\ws\pybind\carma\include\carma/carma/utils.h(34): message : see declaration of 'carma::is_convertible' [D:\ws\pybind\
build\cmake_example.vcxproj]
D:\ws\pybind\carma\include\carma/carma/converters.h(546,77): error C2143: syntax error: missing ';' before '{' [D:\ws\p
ybind\build\cmake_example.vcxproj]
D:\ws\pybind\carma\include\carma/carma/converters.h(546,77): error C2447: '{': missing function header (old-style forma
l list?) [D:\ws\pybind\build\cmake_example.vcxproj]
D:\ws\pybind\carma\include\carma/carma/converters.h(631,15): error C2065: 'detail': undeclared identifier [D:\ws\pybind
\build\cmake_example.vcxproj]
D:\ws\pybind\carma\include\carma/carma/converters.h(631,21): error C4430: missing type specifier - int assumed. Note: C
++ does not support default-int [D:\ws\pybind\build\cmake_example.vcxproj]
D:\ws\pybind\carma\include\carma/carma/converters.h(632,1): error C2146: syntax error: missing ';' before identifier 'N
AMESPACE_END' [D:\ws\pybind\build\cmake_example.vcxproj]`
```
Cmake setup is as follows
```

CMAKE_MINIMUM_REQUIRED(VERSION 3.7)
project(cmake_example)


set(CMAKE_CXX_STANDARD 17)
set(CMAKE_CXX_STANDARD_REQUIRED ON)

add_subdirectory(pybind11)
# add_subdirectory(carma)
include_directories(carma/third_party/armadillo-code/include)
include_directories(carma/include)
link_directories("lib")
pybind11_add_module(cmake_example main.cpp)




```
my lib directories contains blas libs required for armadillo

What's going wrong here?

--- a/include/carma/carma/converters.h
+++ b/include/carma/carma/converters.h
@@ -539,8 +539,8 @@ inline py::array_t<T> to_numpy(arma::Cube<T>&& src, bool copy = true) {
 
 }  // namespace carma
 
-NAMESPACE_BEGIN(pybind11)
-NAMESPACE_BEGIN(detail)
+namespace pybind11 {
+namespace detail {
 
 template <typename armaT>
 struct type_caster<armaT, enable_if_t<carma::is_convertible<armaT>::value>> {
@@ -628,6 +628,6 @@ struct type_caster<armaT, enable_if_t<carma::is_convertible<armaT>::value>> {
 
     PYBIND11_TYPE_CASTER(armaT, _("Numpy.ndarray[") + npy_format_descriptor<T>::name + _("]"));
 };
-NAMESPACE_END(detail)
-NAMESPACE_END(pybind11)
+} /* namespace detail */
+} /* namespace pybind11 */
 #endif /* ARMA_CONVERTERS */

